lib/commands/file-movement.js

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash';
+import log from '../logger';
+import path from 'path';
+import { errors } from 'appium-base-driver';
+import { fs, mkdirp, util, zip } from 'appium-support';
+import { DELETE_FILES_FEATURE } from '../constants';
+
+// List of env variables, that can be expanded in path
+const KNOWN_ENV_VARS = [
+  'APPDATA', 'LOCALAPPDATA',
+  'PROGRAMFILES', 'PROGRAMFILES(X86)',
+  'PROGRAMDATA', 'ALLUSERSPROFILE',
+  'TEMP', 'TMP',
+  'HOMEPATH', 'USERPROFILE', 'PUBLIC'
+];
+const commands = {};
+
+commands.pushFile = async function pushFile (remotePath, base64Data) {
+  if (remotePath.endsWith(path.sep)) {
```

Comment:
I would say it makes sense to change the security feature name to `change_fs` and also add it to this method, since one could also corrupt system files by owerwriting them

Reply:
Fixed.